src/components/DataProvider/index.js

```diff
@@ -560,20 +562,19 @@ DataProvider.propTypes = {
   collections: GriffPropTypes.collections,
   // xSubDomain => void
   onXSubDomainChanged: PropTypes.func,
-  // xDomain => void
-  onXDomainChanged: PropTypes.func,
   opacity: PropTypes.number,
   opacityAccessor: PropTypes.func,
   pointWidth: PropTypes.number,
   pointWidthAccessor: PropTypes.func,
   strokeWidth: PropTypes.number,
+  isXSubDomainSticky: PropTypes.bool,
```

Comment:
Could you put some docs on this and `limitXSubDomain`? I (the user) have no idea what they're supposed to do.

Reply:
Ping